frontend/src/store/slices/map/map.slice.ts

```diff
@@ -0,0 +1,48 @@
+import { AppThunk } from 'common/types';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { ReducerName, MapKey } from 'common/enums';
+import { IMap } from 'common/interfaces';
+import { notification as notificationService, mapApi } from 'services';
+import { HttpError } from 'exceptions';
+
+type MapState = {
+  selectedCoords: IMap;
+};
+
+const initialState: MapState = {
+  selectedCoords: {
+    [MapKey.MAX_LAT]: 0,
+    [MapKey.MAX_LNG]: 0,
+    [MapKey.MIN_LNG]: 0,
+    [MapKey.MIN_LAT]: 0,
+  },
```

Comment:
We don't need this in the slice

Just select coords and send them. That it. Without any savings